### PR TITLE
Nar-Sie will now convert girders to cult girders and metal sheets to runed metal sheets

### PR DESCRIPTION
**Pull request by ChangelingRain**

### CODE REVIEW

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -57,6 +57,10 @@ var/global/list/datum/stack_recipe/metal_recipes = list ( \
 	flags = CONDUCT
 	origin_tech = "materials=1"
 
+/obj/item/stack/sheet/metal/narsie_act()
+	if(prob(20))
+		new /obj/item/stack/sheet/runed_metal(loc, amount)
```
Comment (lordpidey) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: Don't you need a qdel here?

**Comment by Jazaen** [comment]
Could we see sprites?

**Comment by TheRealGLH** [comment]
Aren't these already in the game, though?

File: code/game/objects/structures/girders.dm [hunk 2]
```diff
@@ -280,6 +280,10 @@
 				new remains(loc)
 				qdel(src)
 
+/obj/structure/girder/narsie_act()
+	if(prob(25))
+		new /obj/structure/girder/cult(loc)
+
```
Comment (KorPhaeron) [inline code/game/objects/structures/girders.dm]: And here

File: code/game/objects/items/stacks/sheets/sheet_types.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: oh right

**Comment by Jazaen** [comment]
I retract my point then, since I am apparently unable to even notice that no new sprite files have been changed.

File: code/game/objects/structures/girders.dm [hunk 3]
```diff
@@ -376,9 +381,8 @@
 	else
 		..()
 
-/obj/structure/girder/cult/blob_act()
```
Comment (lordpidey) [inline code/game/objects/structures/girders.dm]: Why remove this?

Comment (ChangelingRain) [inline code/game/objects/structures/girders.dm]: Now that cult girders are a subtype of girders it's duplicated from girder/blob_act()

**Comment by KorPhaeron** [comment]
Pretty sure this fixes at least one issue related to cult girders

File: code/game/objects/structures/girders.dm [hunk 4]
```diff
@@ -280,6 +280,11 @@
 				new remains(loc)
 				qdel(src)
 
+/obj/structure/girder/narsie_act()
+	if(prob(25))
```
Comment (KorPhaeron) [inline code/game/objects/structures/girders.dm]: What is the point of the prob stuff?

Comment (ChangelingRain) [inline code/game/objects/structures/girders.dm]: GRADUAL CONVERSION

**Comment by Armhulen** [comment]
I really like these

**Pull request merged**